ManatiCR/manatime
# 14: Content type tasks.

**Steps to test**
- See if the features manatime_feature_tasks was added and activated.
- Add a new tasks node.

**DoD**
- As an administrator I should be able to go to node/add/task and add a new task with the described fields
- As an administrator, I should be able to view a task (no themed) and click on the Edit tab and update that task.

[issue link](https://github.com/ManatiCR/manatime/issues/14)
